elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -167,4 +179,24 @@ protected void convertAttribute(PathAddress address, String attributeName, Model
         }
     };
 
+    private static final RejectAttributeChecker REJECT_FLUSHED_DIFFERENT_FROM_SYNCHRONIZED = new RejectAttributeChecker.DefaultRejectAttributeChecker() {
```

Comment:
<minor>should be `REJECT_IF_DIFFERENT_FROM_SYNCHRONIZED` to be consistent with `DISCARD_IF_EQUALS_SYNCHRONIZED `</minor>